mavros/src/plugins/setpoint_attitude.cpp

```diff
@@ -106,7 +106,10 @@ class SetpointAttitudePlugin : public plugin::PluginBase,
 	std::string tf_frame_id;
 	std::string tf_child_frame_id;
 
+	message_filters::Subscriber<mavros_msgs::Thrust> sub;
```

Comment:
General "sub" bad for later support, thats why i want more specific name like `thd_sub`.
And to solve problem with `tf_start()`, just pass by reference.

Or move whole `tf_start()` code out of mixin, since it looks specific to that plugin.
Or you plan to use that variation somewhere else?